third_party/move/move-vm/runtime/src/storage/ty_layout_converter.rs

```diff
@@ -103,6 +106,120 @@ where
         self.struct_definition_loader.runtime_environment()
     }
 
+    pub(crate) fn contains_option_type(
+        &self,
+        ty: &Type,
+        gas_meter: &mut impl DependencyGasMeter,
+        traversal_context: &mut TraversalContext,
+    ) -> PartialVMResult<bool> {
+        let option_module_id =
```

Comment:
Its better to make these static const? Otherwise these are re-created on every recursive call.

Reply:
done